### ISSUE TITLE
Currently, the search function in a chat is not accessible

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2134]
Here is what I get when searching:

* A text field
* A "filter by user" button (maybe the translation is wrong but you get the idea)
* Two unlabeled buttons

And that's all. I don't know how search results work. Perhaps they work like moving focus to the result, but focus is unreliable, as discussed in #1557 and #1183 . I don't know what the unlabeled buttons do, either.

If I press filter by user, I get:

* A text field
* Nothing (literally the next tab press NVDA gets silent)
* An unlabeled button
* User information, but in json
